GovUk.Frontend.Umbraco.ExampleApp/Models/ModelsBuilder/GovukAccordionSettings.generated.cs

```diff
@@ -20,7 +20,7 @@ namespace Umbraco.Cms.Web.Common.PublishedModels
 {
 	/// <summary>Accordion settings</summary>
 	[PublishedModel("govukAccordionSettings")]
-	public partial class GovukAccordionSettings : PublishedElementModel, IGovukCssClasses, IGovukGrid, IGovukGridColumnClasses, IGovukHeadingLevel
+	public partial class GovukAccordionSettings : PublishedElementModel, IGovukCssClasses, IGovukGrid, IGovukGridColumnClasses
```

Comment:
We seem to have lost the ability to customise the heading level of an accordion, was this intentional?